source/agora/node/Registry.d

```diff
@@ -64,13 +71,15 @@ public class NameRegistry: NameRegistryAPI
     private ValidatorInfo[] validator_info;
 
     ///
-    public this (string realm, RegistryConfig config, FullNodeAPI agora_node)
+    public this (string realm, RegistryConfig config, FullNodeAPI agora_node, ManagedDatabase cache_db)
```

Comment:
```suggestion
    public this (string realm, RegistryConfig config, FullNodeAPI agora_node,
        ManagedDatabase cache_db)
```
It exceeds the limit of 80 characters in a line. We do not strictly force 80 characters per line but we recommend it. So if adding a newline doesn't make the code ugly, it would be good to add a newline.